common/src/main/java/org/apache/celeborn/common/network/server/TransportRequestHandler.java

```diff
@@ -73,8 +76,64 @@ public void channelInactive() {
 
   @Override
   public void handle(RequestMessage request) {
+    logger.trace("Received request {} from {}", request.getClass().getName(), reverseClient);
```

Comment:
how much cost of `request.getClass().getName()`? should we wrap this line with if `logger.isTraceEnabled`?

Reply:
`request.getClass().getName()` is a relatively lightweight operation but this is going to be done very frequently, so I can wrap it up.